protos/capture.proto

```diff
@@ -218,3 +218,25 @@ message CaptureHeader {
   uint32 num_timers = 2;
   uint32 version = 3;
 }
+
+message TimerData {
+  uint64 start = 1;
+  uint64 end = 2;
+  int32 process_id = 3;
+  int32 thread_id = 4;
+  uint32 depth = 5;
+
+  enum Type {
+    NONE = 0;
```

Comment:
kNone here and below: https://google.github.io/styleguide/cppguide.html#Constant_Names